trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Lift/vortex_lift.py

```diff
@@ -62,11 +63,10 @@ def vortex_lift(state,settings,geometry):
                 gamma      = wing.sweeps.leading_edge
                 
             AR = wing.aspect_ratio
-            a = AoA[Mc < 1.0]
-            
+            a = np.abs(AoA[Mc < 1.0])
+            signs = np.sign(AoA[Mc < 1.0])
```

Comment:
Should this be based on AoA and incidence angles combined? I could see many cases where it flips too early.

Reply:
Sure will do.